File: sct_source/SourceCode/API_SoCongThuong/Models/CateRPTurnOverIndustAncolModel.cs
using System;
using System.Collections.Generic;

namespace API_SoCongThuong.Models
{
    public partial class CateReportTurnOverIndustAncolModel
    {
        public Guid CateReportTurnOverIndustAncolId { get; set; }
        /// <summary>
        /// 1: Buôn rượu 2: Bán lẻ
        /// </summary>
        public Guid BusinessId { get; set; }
        public string? AlcoholBusinessName { get; set; }
        public string? Address { get; set; }
        public string? PhoneNumber { get; set; }
        public string? LicenseCode { get; set; }
        public DateTime? LicenseDate { get; set; }
        public string? LicenseDateDisplay { get; set; }
        public long QuantityBoughtOfYear { get; set; }
        public long TotalPriceBoughtOfYear { get; set; }
        public long QuantitySoldOfYear { get; set; }
        public long TotalPriceSoldOfYear { get; set; }
        public int YearId { get; set; }
        /// <summary>
        /// 1: Hoạt động 0: Không hoạt động
        /// </summary>
        public bool? IsAction { get; set; }
        /// <summary>
        /// 1: Đã xóa; 0: Chưa xóa
        /// </summary>
        public bool IsDel { get; set; }
        /// <summary>
        /// Thời gian tạo
        /// </summary>
        public DateTime CreateTime { get; set; }
        public string CreateTimeDisplay { get; set; } = "";
        /// <summary>
        /// Người tạo
        /// </summary>
        public Guid CreateUserId { get; set; }
        public string CreateName { get; set; } = "";
        public DateTime? UpdateTime { get; set; }
        public Guid? UpdateUserId { get; set; }
    }
}

File: sct_source/SourceCode/FE_SoCongThuong/Angular/src/app/modules/app-pages/mnfvspproducingalcohol-page/_models/caterpturnoverindustancol.model.ts
import { BaseModel } from '../../../../_metronic/shared/crud-table/models/base.model';

export interface CateRPTurnOverIndustAncolModel extends BaseModel {
  cateReportTurnOverIndustAncolId: string,
  businessId: string,
  quantityBoughtOfYear: number | null, //Số lượng mua trong năm
  totalPriceBoughtOfYear: number | null, //Tổng giá trị mua trong năm
  quantitySoldOfYear: number | null, //Số lượng bán trong năm
  totalPriceSoldOfYear: number | null, //Tổng giá trị bán trong năm
  yearId: any
}
